src/openforms/forms/tests/test_formdefinition_admin.py

```diff
@@ -26,3 +30,33 @@ def test_formdefinition_list_for_user_with_all_perms(self):
         response = self.app.get(url, user=user)
 
         self.assertEqual(response.status_code, 200)
+
+    @override_settings(LANGUAGE_CODE="en")
+    def test_copy_with_long_names_succeeds(self):
+        user = SuperUserFactory.create()
+        FormDefinitionFactory.create(
+            name="This is a really long long name for formdefinition",
+            internal_name="This is a really long long name for formdefinition",
+        )
+
+        self.assertEqual(FormDefinition.objects.count(), 1)
+
+        url = reverse("admin:forms_formdefinition_changelist")
+        response = self.app.get(url, user=user)
+        form = response.forms["changelist-form"]
+        form["action"] = "make_copies"
+        form["_selected_action"].checked = True
+        response = form.submit("submit", value="0")
+
+        form_definition_copy = FormDefinition.objects.last()
```

Comment:
I think this could cause flaky tests when the database doesn't return the items in the same order. Usually we fix it by adding `.order_by("pk")` and then taking the last item.